code/modules/vore/eating/bellymodes_vr.dm

```diff
@@ -472,5 +472,7 @@
 			put_in_egg(P,1)
 
 	if(play_sound)
+		for (var/mob/living/M in contents)
+			M << sound(play_sound)
```

Comment:
You can use M.playsound_local(loc, play_sound) here

Reply:
Dunno I feel like something's broke the get_turf which now is get_step instead which from inside the belly obj pointed to nullspace instead or someshit and that's what I think was the reason for the prey not hearing them at all after the update changed it from direct to local. Found out the old get_turf that actually did account for the locs var was scrapped nearly 2 years ago or something.

Reply:
Yeah didn't work. Testing with M.loc now

Reply:
weeeird. 

`M.playsound_local(get_turf(M), play_sound, 65)` should work

Reply:
bah kinda redundant anyway as all it would add would be distance/volume/falloff calculations for a prey that would literally be inside the thing in any case before doing the << thing anyway at the end

Reply:
playsound_local can be set to no falloff. also if the prey is literally on top of the location, then falloff can't occur because they're on top of the source. 

But yee, the << will work just as well.

Reply:
<< will work, but IIRC, it's not the desired means of doing it.

@Arokha have any better ideas?

Reply:
That's pretty much what everything leads to.
https://github.com/VOREStation/VOREStation/blob/master/code/game/sound.dm#L90